sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubConnectionString.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service.Authentication
+{
+    internal class IotHubConnectionString
+    {
```

Comment:
Add unit tests for custom logic.

Reply:
Yes, I will take that up as a subsequent PR. I wanted to get this PR out first, so that others can be unblocked from writing tests and samples.

Reply:
Is there a task for unit tests on the board?